agent/metadata_service.go

```diff
@@ -132,7 +130,7 @@ func (mds *metadataService) getCredentials(w http.ResponseWriter, r *http.Reques
 		Code:            "Success",
 		LastUpdated:     time.Now().UTC().Format(time.RFC3339),
 		Type:            "AWS-HMAC",
-		AccessKeyId:     creds.AccessKeyId,
+		AccessKeyID:     creds.AccessKeyId,
```

Comment:
Thanks, that's a fair point, and it highlights the reason we were doing these commits in the first place, which is to ensure that the code is easy to work in and testing is comprehensive. In this particular case, we should have been enforcing the format of the json we're outputting (as it's an interface that we need to adhere to), and our tests should include that we're producing the expected json output, which is obviously something we skipped here.